common/src/main/java/com/wynntils/functions/CharacterFunctions.java

```diff
@@ -87,6 +87,40 @@ public Double getValue(FunctionArguments arguments) {
         }
     }
 
+    public static class StatusEffectsFunction extends Function<String> {
+        @Override
+        public String getValue(FunctionArguments arguments) {
+            List<String> statusEffectsList = Models.StatusEffect.getStatusEffects().stream()
+                    .map(statusEffect -> statusEffect.asString().getString())
+                    .toList();
+
+            return String.join("\n", statusEffectsList);
+        }
+    }
+
+    public static class StatusEffectActiveFunction extends Function<Boolean> {
+        @Override
+        public Boolean getValue(FunctionArguments arguments) {
+            String query = arguments.getArgument("query").getStringValue();
+            return !Models.StatusEffect.getStatusEffects().stream()
+                    .filter(statusEffect ->
+                            statusEffect.getName().getStringWithoutFormatting().equals(query))
+                    .toList()
+                    .isEmpty();
```

Comment:
One final change :) You can use anyMatch here, so you don't need to do ! and .isEmpty. (also it's slightly faster).

Reply:
queued